docs/img-src/action-sequence.mmd

```diff
@@ -0,0 +1,19 @@
+sequenceDiagram
+  participant Test Runner(1)
+  participant Detox(3)
+  participant Detox Server(5)
+  participant Detox Client(7)
+  participant Earl Grey(9)
+
+  %% Note over Test Runner(1),Detox Server(5): Runs on your computer
+  %% Note over Detox Client(7),Earl Grey(9): Runs on the device
+
+  Test Runner(1)->>Detox(3): assertion / action (2)
+  Detox(3)->>Detox Server(5): native code as JSON (4)
```

Comment:
Can you explain that a bit? would you name Test Runner, Detox, etc only in the text? I would find that a bit confusing if you try to get a general overview of what exactly is happening 👍